environment_dev.yml

```
@@ -6,8 +6,9 @@ channels:
   - conda-forge
 dependencies:
   - python=3.10
-  - yarn>=4.4.0
-  - nodejs>=18.0
+  # Match the version of Node.js defined in the Dockerfile.
+  # Also, do not install Yarn separately, use Corepack (comes with Node.js).
+  - nodejs=22.8.0
```

Do we want to have a hard pin on the node version? This means the environment would need to be updated quite often and IMO this is not needed. We could pin to the major instead
```suggestion
  - nodejs >=22.0
```

---

Thanks for the suggestion! Let's give it a try and see.

You have much more experience than I do about how to manage environments, so I lean on your expertise here :)